pyproject.toml

```
@@ -17,7 +17,8 @@ dependencies = [
   "numpy >=1.24",
   "astropy >=6.0.0",
   # "rad >=0.25.0",
-  "rad @ git+https://github.com/spacetelescope/rad.git",
+  # "rad @ git+https://github.com/spacetelescope/rad.git",
+  "rad @ git+https://github.com/WilliamJamieson/rad.git@remove_internal_tags",
```

```suggestion
  "rad @ git+https://github.com/spacetelescope/rad.git",
```
Reminder to update prior to merging.